src/System.Management.Automation/engine/parser/AstVisitor.cs

```diff
@@ -425,7 +425,7 @@ internal static bool Contains(Ast ast, Func<Ast, bool> predicate, bool searchNes
 
             var searcher = new AstSearcher(predicate, stopOnFirst: true, searchNestedScriptBlocks: searchNestedScriptBlocks);
             ast.InternalVisit(searcher);
-            return searcher.Results.Any();
+            return searcher.Results.Count > 0;
```

Comment:
We've been here in  #12956 :-)

Reply:
Yes, this PR is follow-up for some missed cases.